### PR TITLE
Add a docker instructions for easy development/testing

### PR DESCRIPTION
**Pull request by akofink**
Use the instructions in the Dockerfile to test!

Note: `rm Gemfile.lock` first

### CODE REVIEW

**Comment by ehelms** [comment]
Looks good but I am curious what this adds over simply adding to the README the command to run the Jekyll container directly? The intermediate container doesn't seem to add much value?

**Comment by akofink** [comment]
:rofl: didn't think of that! I will simply add some docs to the Readme.

**Comment by jturel** [comment]
Good point @ehelms. This did work nicely though :)

**Comment by akofink** [comment]
Updated!

**Comment by ehelms** [comment]
Looks good... final question, is there value in adding a Rake task to run what's in the README? Something like:

```
rake docker:serve
```

**Comment by akofink** [comment]
No way! ;) If I use rake, I have to install ruby + gems, and that's the whole point of using Docker.

**Comment by jturel** [comment]
Works for me - thanks!

**Review by jturel — APPROVED** [review]

**Comment by ehelms** [comment]
Fair enough, you should only need ruby + rake but it was worth asking :)

**Review by ehelms — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -19,6 +19,14 @@ to generate your site in the \_site directory, or
 
 To start Jekyll server locally.
 
+## With Docker
```
Comment (sean797) [inline README.md]: Shouldn't this be a subheading of  `## Testing locally` above?

Comment (akofink) [reply README.md]: This is running *not* locally but in a container, so I don't think it makes sense under that header. Also this isn't for running the tests (though I guess you could), it's for running the jekyll app for manual testing.

Comment (sean797) [reply README.md]: right okay, then I think either of the following would make better sense.

## Testing 

### Locally

### With Docker

---------------------

## Testing locally

## Testing with Docker

Comment (akofink) [reply README.md]: But as I said, that header is for running the automated tests, not for running the app. If I were to move it under a top level header, I'd move it to the contributing section. Eh?

Comment (sean797) [reply README.md]: There are no automated tests. I'm fairly sure what you added is the same running `rake server`

Comment (akofink) [reply README.md]: Doh! You're right :) Ok, I'll put it under the testing section.

**Comment by akofink** [comment]
Updated

**Review by sean797 — APPROVED** [review]

**Review by stbenjam — APPROVED** [review]
I think everyone's happy now, so merging.

**Pull request merged**